### ISSUE TITLE
Download does not work on Windows while using VPN service

### ISSUE THREAD

**Issue by rezyh** [AM2R-Community-Developers/AM2RLauncher#70]
On Windows 10 I use a VPN service.  I would select "Download" and it would not work.

Checked in the log file and found these two lines:

2024-02-28 16:33:32,566 [1] INFO AM2RLauncher.MainForm - Attempting to clone repository https://github.com/AM2R-Community-Developers/AM2R-Autopatcher-Windows.git...
2024-02-28 16:34:44,498 [1] ERROR AM2RLauncher.MainForm - LibGit2SharpException: failed to read data: The server returned an invalid or unrecognized response

Figured it might be because I use a VPN.  Turned it off and the program worked.

Just thought I would give you the heads up, not sure if there is much you can fix code wise, but maybe an FAQ section?  Hope this is helpful and not annoying, Thanks!

**Comment by Miepee** [comment]
Based on 
> The server returned an invalid or unrecognized response

This looks more like a github issue than us. Can you try whether selecting the mirror to gitlab works through your VPN?

**Comment by Miepee** [comment]
@rezyh can you confirm the above theory?

**Comment by rezyh** [comment]
@Miepee Sorry I missed your previous comment.  I just went back and tried to duplicate the issue I was having, and it is now working just fine.  I tried multiple configurations, but it works fine with both the mirror and primary.  I think your suspicions are correct that it wasn't an issue with your program.  More likely is I had some strange settings with my VPN that needed to be reset. Thanks for taking the time to look into it regardless.
